ui/selectors/selectors.js

```diff
@@ -78,6 +78,16 @@ export function getCurrentKeyring(state) {
   return keyring;
 }
 
+/**
+ * Checks if the current wallet is a hardware wallet.
+ * @param {Object} state
+ * @returns {Boolean}
+ */
+export function isHardwareWallet(state) {
+  const keyring = getCurrentKeyring(state);
+  return keyring.type.search('Hardware') !== -1;
```

Comment:
Probably my own bias but can we use `startsWith` or `includes` here?

Reply:
For sure. I found this code in our codebase, but I like `includes` more as well.